security/patch-per-service-tls.yml

```diff
@@ -0,0 +1,247 @@
+        kubernetes:
+          volumes:
+          - id: orca-cert
+            type: secret
+            mountPath: /orca-cert
+          - id: truststore
+            type: secret
+            mountPath: /truststore
+      echo:
+        # Mount the keystore and truststore secrets to the container
+        kubernetes:
+          volumes:
+          - id: echo-cert
+            type: secret
+            mountPath: /echo-cert
+          - id: truststore
+            type: secret
+            mountPath: /truststore
+      fiat:
+        # Mount the keystore and truststore secrets to the container
+        kubernetes:
+          volumes:
+          - id: fiat-cert
+            type: secret
+            mountPath: /fiat-cert
+          - id: truststore
+            type: secret
+            mountPath: /truststore
+      gate:
+        # Mount the keystore and truststore secrets to the container
+        kubernetes:
+          volumes:
+          - id: gate-cert
+            type: secret
+            mountPath: /gate-cert
+          - id: truststore
+            type: secret
+            mountPath: /truststore
+      igor:
+        # Mount the keystore and truststore secrets to the container
+        kubernetes:
+          volumes:
+          - id: igor-cert
+            type: secret
+            mountPath: /igor-cert
+          - id: truststore
+            type: secret
+            mountPath: /truststore
+      kayenta:
+        # Mount the keystore and truststore secrets to the container
+        kubernetes:
+          volumes:
+          - id: kayenta-cert
+            type: secret
+            mountPath: /kayenta-cert
+          - id: truststore
+            type: secret
+            mountPath: /truststore
+      rosco:
+        # Mount the keystore and truststore secrets to the container
+        kubernetes:
+          volumes:
+          - id: rosco-cert
+            type: secret
+            mountPath: /rosco-cert
+          - id: truststore
+            type: secret
+            mountPath: /truststore
+      dinghy:
+        # Mount the certs to the container.  The Truststore is not needed as this is a GoLang Service.
+        kubernetes:
+          volumes:
+          - id: dinghy-cert
+            type: secret
+            mountPath: /dinghy-cert
+      terraformer:
+        # Mount the certs to the container.  The Truststore is not needed as this is a GoLang Service.
+        kubernetes:
+          volumes:
+          - id: terraformer-cert
+            type: secret
+            mountPath: /terraformer-cert
+    profiles:
+      front50:
+        server:
+          ssl:
+            # Enable TLS and set the keystore, certs, and trustore for the Server component of the service
+            enabled: true
+            key-store: /front50-cert/keystore.jks
+            key-store-type: jks
+            key-store-password: changeit # This should match the password set when the keystore was created
```

Comment:
Can you add all secrets to `secrets-example.env` and reference them here as kubernetes secrets? Since these are documentation and example configs we try to put all best practices in place like secret handling. They can just be one secret for all keystores and one for the truststore.

Reply:
Yeah, let's add them to the secrets-example.env just for consistency.

Reply:
Fixed.